DEVELOPER_GUIDE.md

```diff
@@ -34,8 +34,8 @@ If you use pyenv:
 
 ```bash
 cd libs/libcommon/
-pyenv install 3.9.15
-pyenv local 3.9.15
+pyenv install 3.9.18
+pyenv local 3.9.18
```

Comment:
Sorry! I just saw it now...

Reply:
Yes, I think the developer guide is OK. There is no section for "update to a new version of python" but I'm not sure we need it